lua/gruvbox-baby/config.lua

```diff
@@ -22,6 +22,7 @@ config = {
   string_style = opt("string_style", "nocombine"),
   variable_style = opt("variable_style", "NONE"),
   highlights = opt("highlights", {}),
+  overrides = opt("overrides", {}),
   telescope_theme = opt("telescope_theme", false),
```

Comment:
I think `palette` could be a better name for this option, or `color_overrides` ?

Reply:
imo palette is the built-in palette, and color_overrides seems to make more sense.

Reply:
or even `palette_overrides`

Reply:
idk, you're the owner, you should choose.

Reply:
I prefer `color_overrides`, but if you put `palette_overrides` is cool as well.

Reply:
alright then, `color_overrides` it is